.trivyignore

```diff
@@ -0,0 +1,2 @@
+# libssl3
+CVE-2023-5678 exp:2023-12-31
```

Comment:
![image](https://github.com/hypertrace/hypertrace-ingester/assets/40003481/51ae4628-86ec-44df-963d-6922056a0f25)
didn't report under PR tab but guessing this is the cause (based on other repo observations)